misk-aws/src/test/kotlin/misk/dynamodb/DockerDynamoDb.kt

```diff
@@ -0,0 +1,81 @@
+package misk.dynamodb
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.client.builder.AwsClientBuilder
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import com.github.dockerjava.api.model.ExposedPort
+import com.github.dockerjava.api.model.Ports
+import misk.containers.Composer
+import misk.containers.Container
+import misk.logging.getLogger
+import misk.testing.ExternalDependency
+
+/**
+ * A test DynamoDb Service. Tests can connect to the service at 127.0.0.1:[clientPort]
+ */
+internal object DockerDynamoDb : ExternalDependency {
+
+  private val log = getLogger<DockerDynamoDb>()
+  private const val clientPort = 8000
```

Comment:
I'm confused. Is this the internal docker port or the port we use to connect? I'm not sure it is a great default.

Reply:
Ooohhh good point. It'd be nice if we could have Java auto-assign the port, but that might make it difficult to make this shared across tests. Either way, 8000 is probably a port you'll run into conflicts on :). I think this is currently both the external and internal port

Reply:
It is both internal and external. I'm actually having trouble remapping it :(

Reply:
I think you can NOT have port 8000 :-) It's taken :-)

Reply:
It worked for me 

This file will be removed